python/setup.py

```diff
@@ -167,7 +167,6 @@ def _supports_symlinks():
                   'pyspark.ml',
                   'pyspark.ml.linalg',
                   'pyspark.ml.param',
-                  'pyspark.ml.stat',
```

Comment:
I think it was. There is `pyspark.mllib.stat` but no `pyspark.ml.stat`. Seems it is mistakenly missing the non-parity between `mllib` and `ml` sub-modules.